docs/.c4/views/overviews/overview.views.c4

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2025 Defense Unicorns
+ * SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+ */
+
+ views "uds-core-overviews" {
+  view index of cloud.cluster.uds-core {
+    title "Base UDS Core Overview"
+
+    include *
+  }
+
+  view uds-core-namespaces of cloud {
+    title "Namespace Architecture Overview"
+
+    group {
+      border solid
+      color background
+      opacity 100%
+
+      // include all elements inside uds-core
+      include
+        *,
+        cluster.uds-core.*,
+        cloud.cluster.external-services.*
```

Comment:
If these external services could be arranged top - down that would make this one less wide and more appealing 😅 (recognize we might not have control here).